index.js

```diff
@@ -5,3 +5,6 @@ module.exports.Chart = excelwriter.Chart;
 module.exports.Color = excelwriter.Color;
 module.exports.Format = excelwriter.Format;
 module.exports.Workbook = excelwriter.Workbook;
+module.exports.ValidationType = excelwriter.ValidationType;
+module.exports.ValidationCriteria = excelwriter.ValidationCriteria;
+module.exports.ValidationErrorType = excelwriter.ValidationErrorType;
```

Comment:
```suggestion
module.exports.DataValidation = excelwriter.DataValidation;
```

One export, and all the enum values should be attached to it, like I do for `Format`.

Reply:
Actually, as you say it might not make sense to have this as a class. Ignore this for now.